Tabloid/Repositories/CategoryRepository.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Tabloid.Models;
+
+namespace Tabloid.Repositories
+{
+    public class CategoryRepository : BaseRepository, ICategoryRepository
+    {
+        public CategoryRepository(IConfiguration config) : base(config) { }
+        public List<Category> GetAll()
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = "SELECT id, name FROM Category Order By Name";
+                    var reader = cmd.ExecuteReader();
+
+                    var categories = new List<Category>();
+
+                    while (reader.Read())
+                    {
+                        categories.Add(new Category()
+                        {
+                            Id = reader.GetInt32(reader.GetOrdinal("Id")),
```

Comment:
Are there DbUtils methods that could be used in this repo to do some of this work? We'll want to use those utils methods consistently across the codebase.